client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -186,13 +164,10 @@ function getCurrentPaymentMethodId( payment ) {
 }
 
 function getChangePaymentMethodTitleCopy( currentPaymentMethodId ) {
-	if ( isEnabled( 'purchases/new-payment-methods' ) ) {
-		if ( [ 'credits', 'none' ].includes( currentPaymentMethodId ) ) {
-			return titles.addPaymentMethod;
-		}
-		return titles.changePaymentMethod;
+	if ( [ 'credits', 'none' ].includes( currentPaymentMethodId ) ) {
+		return titles.addPaymentMethod;
 	}
-	return titles.editCardDetails;
```

Comment:
I think `titles.editCardDetails` is only used in one other place. Maybe we can remove it entirely? (There may be other titles we can remove with this PR too.)